assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -566,6 +567,11 @@ export default function AudienceTiles( { Widget, widgetLoading } ) {
 							isPartialData,
 						} = getAudienceTileData( audienceResourceName, index );
 
+						// Filter (not set) value from the top countries report if present.
+						const filteredTopCitiesRows = topCities?.rows
+							? reportRowsWithSetValues( topCities.rows )
+							: {};
```

Comment:
Shouldn't this variable be an Array (of rows)? Trying to understand why the fallback would be an object.

Reply:
Yes, this needs to be an array. Changed it in [this commit](https://github.com/google/site-kit-wp/pull/9984/commits/013a67b853dda930ccd20db57cecc72d7535f7f7).